### PR TITLE
feat: parser address integration

### PR DESCRIPTION
**Pull request by ssamkkim**
- (hopefully) fixed my crlf/lf eslint issues each time I sync up
- added (test) ability for parseTransaction functions to take in an address and spit out specific actiontypes depending on it (makes it cleaner on the UI code side)
- moved all (necessary) parser functions to that standard 
- fixed transfer parser to be like how the unknown is (with ignoring native transfers that are the rent fees)
- highered rent requirements for the util functin

breaking: 
- for testing purposes, the queries.ts passes in the address I used to test the address integration, should actually be the params address when in address view (or address connected to a tx page)

### CODE REVIEW

**Comment by ssamkkim** [comment]
the eslint fix I'm trying to apply + the one that github tests are failing me on (atm):
https://stackoverflow.com/questions/53516594/why-do-i-keep-getting-eslint-delete-cr-prettier-prettier

File: apps/web/src/lib/state/queries.ts [hunk 1]
```diff
@@ -34,7 +34,10 @@ export const solanaAccountInfo = {
 export const solanaTransactions = {
     formatter: (data: any) =>
         data.map((tx: EnrichedTransaction) => ({
-            parsed: parseTransaction(tx),
+            parsed: parseTransaction(
+                tx,
+                "BGkpRfK8L5Yy7A8wimA4CXra2GhFxdSWK944Ap6dPKCu"
+            ),
```
Comment (qudo-code) [inline apps/web/src/lib/state/queries.ts:40]: Perfect for now while we merge.

File: packages/xray-proton/src/parsers/unknown.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { LAMPORTS_PER_SOL } from "@solana/web3.js";
 import type { EnrichedTransaction, Source, TokenTransfer } from "helius-sdk";
-import { ProtonTransaction, ProtonTransactionAction } from "../types";
+import { ProtonTransaction, ProtonTransactionAction, SOL } from "../types";
```
Comment (qudo-code) [inline packages/xray-proton/src/parsers/unknown.ts:3]: Would be cool if we could reverse lookup this from namor hm

**Review by qudo-code — APPROVED** [review]

File: packages/xray-proton/src/parsers/unknown.ts (hunk 2 above)
Comment (ssamkkim) [reply packages/xray-proton/src/parsers/unknown.ts:3]: wdym?

File: .eslintrc [hunk 3]
```diff
@@ -1,4 +1,10 @@
 {
-    "root"    : true,
-    "extends" : [ "custom" ]
-}
\ No newline at end of file
+    "extends": ["custom"],
+    "prettier/prettier": [
```
[the remark this answers is not in the document]
Comment (ssamkkim) [reply .eslintrc:3]: oh so just change the eslint-config-custom packages and erase the changes in the other eslintrc files

**Pull request merged**